ST_stats.c

```diff
@@ -1480,18 +838,18 @@ static void _make_header_with_std( char *buf) {
 }
 
 /***********************************************************/
-static void _make_header( char *buf) {
+void make_header( char *buf) {
```

Comment:
Function is never used.

Reply:
@fdpierson I think we should leave these functions in for now just in case, but I will open an issue so we remember to address unused functions in another branch.

Reply:
I created issue #393 to document these functions.